support-frontend/assets/pages/[countryGroupId]/checkout.tsx

```diff
@@ -468,6 +486,9 @@ function CheckoutComponent({ geoId }: Props) {
 	const [isProcessingPayment, setIsProcessingPayment] = useState(false);
 	const formOnSubmit = async (formData: FormData) => {
 		setIsProcessingPayment(true);
+		if (!billingState) {
+			setBillingState('');
+		}
```

Comment:
I'm a little confused by this code as it's been accessed in a few places

From my reading - we have two states:
- `billingState` is not required
- `billingState` is required
  - `billingState` is invalid if `''`
  - otherwise `billingState` is valid 🥳 

I wonder if there is a clearer way to indicate this in the code?

Reply:
Its possible where the `billingState` starts at undefined and you directly press 'Pay Now' to run the `formSubmit`. We would want the error message to be displayed in this scenario.  The 'formOnSubmit` checks for an undefined and, if found, sets the `billingState` to '' to display the error message.